package.json

```diff
@@ -14,7 +14,7 @@
 		"node": ">=12"
 	},
 	"scripts": {
-		"test": "xo && nyc ava",
+		"test": "xo && nyc ava && markdownlint '**/*.md'",
```

Comment:
FYI this doesn't work with Windows native CMD; double quotes work.

Reply:
https://github.com/sindresorhus/eslint-plugin-unicorn/pull/1604#discussion_r747903908